test.py

```diff
@@ -1,4 +1,5 @@
 from test1 import func
 
 print("my")
-print("world")
```

Comment:
Printing to standard output is not ideal for automated testing. Replace print statements with assertions to verify the functionality of `func`.

```suggestion
assert func() == "expected_result"
```